sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/signaling/CommunicationListener.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.chat.signaling;
+
+import com.azure.android.core.logging.ClientLogger;
+import com.microsoft.trouterclient.ITrouterConnectionInfo;
+import com.microsoft.trouterclient.ITrouterListener;
+import com.microsoft.trouterclient.ITrouterRequest;
+import com.microsoft.trouterclient.ITrouterResponse;
+
+import org.json.JSONObject;
+
+final class CommunicationListener implements ITrouterListener {
+
+    private final ClientLogger logger;
+    private final String chatEventId;
+    private final RealTimeNotificationCallback listenerFromConsumer;
+
+    CommunicationListener(String chatEventId, RealTimeNotificationCallback listener) {
```

Comment:
Why not use the enum for ChatEventId here instead of string? MakeChatEventId extension of ExpandableStringEnum if necessary.